configure.ac

```diff
@@ -6166,6 +6188,9 @@ PY_STDLIB_MOD([_sha3], [test "$with_builtin_sha3" = yes])
 PY_STDLIB_MOD([_blake2], [test "$with_builtin_blake2" = yes])
 
 PY_STDLIB_MOD([_decimal], [], [], [$LIBMPDEC_CFLAGS], [$LIBMPDEC_LDFLAGS])
+PY_STDLIB_MOD([nis],
+  [], [test "$have_nis" = yes -a "$ac_cv_header_rpc_rpc_h" = yes],
```

Comment:
AFAICS, header check for `rpc/rpc.h` is missing.

Reply:
It's right there:

```
AS_VAR_IF([have_nis], [yes], [
  WITH_SAVE_ENV([
    CPPFLAGS="$LIBNSL_CFLAGS $CFLAGS"
    AC_CHECK_HEADERS([rpc/rpc.h])
  ])
])
```